tools/benchmark/CHANGELOG.md

```diff
@@ -5,3 +5,4 @@
 ### ⚠ BREAKING CHANGES
 
 Update `typescript` dependency from `4.x` to `5.x`.
+Update `version` from `0.50.0` to `0.51.1`.
```

Comment:
This is inside of the `## 0.50.0` section of this file. If you are going to release a new version, it should get its own section in this file, and it should document what changes you are making in that version other than just the version change.